### ISSUE TITLE
pygeoapi_master does not start after upgrade to 0.15.0

### ISSUE THREAD

**Issue by justb4** [geopython/demo.pygeoapi.io#43]
```
docker logs --follow pygeoapi_master
START /entrypoint.sh
Trying to generate openapi.yml
Traceback (most recent call last):
  File "/usr/local/bin/pygeoapi", line 33, in <module>
    sys.exit(load_entry_point('pygeoapi', 'console_scripts', 'pygeoapi')())
  File "/usr/lib/python3/dist-packages/click/core.py", line 1128, in __call__
    return self.main(*args, **kwargs)
  File "/usr/lib/python3/dist-packages/click/core.py", line 1053, in main
    rv = self.invoke(ctx)
  File "/usr/lib/python3/dist-packages/click/core.py", line 1659, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/lib/python3/dist-packages/click/core.py", line 1659, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/lib/python3/dist-packages/click/core.py", line 1395, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/lib/python3/dist-packages/click/core.py", line 754, in invoke
    return __callback(*args, **kwargs)
  File "/usr/lib/python3/dist-packages/click/decorators.py", line 26, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/pygeoapi/pygeoapi/openapi.py", line 1360, in generate
    content = generate_openapi_document(config_file, format_)
  File "/pygeoapi/pygeoapi/openapi.py", line 1335, in generate_openapi_document
    content = yaml.safe_dump(get_oas(s), default_flow_style=False)
  File "/pygeoapi/pygeoapi/openapi.py", line 1292, in get_oas
    return get_oas_30(cfg)
  File "/pygeoapi/pygeoapi/openapi.py", line 1132, in get_oas_30
    p = load_plugin('process', v['processor'])
  File "/pygeoapi/pygeoapi/plugin.py", line 113, in load_plugin
    class_ = getattr(module, classname)
AttributeError: module 'pygeometa.pygeoapi_plugin' has no attribute 'PygeometaMetadataSchemasProcessor'. Did you mean: 'PygeometaMetadataGenerateProcessor'?
ERROR: openapi.yml could not be generated ERROR
```

## Analysis
Almost sure this is due to that `pygeometa`  v0.14.0 is installed, but `pygeometa.pygeoapi_plugin.PygeometaMetadataSchemasProcessor` is not yet in that/any version, only in `main` branch.
So commenting out in `local.config.yml` will fix.

**Comment by justb4** [comment]
Fixed, @tomkralidis can comment-in with next version pygeometa.

**Comment by tomkralidis** [comment]
Thanks @justb4.  pygeometa 0.15.0 now released, uncommented back in 90cafbf53b98941107feddeb3905256303781992.
